setup.py

```diff
@@ -1,47 +1,33 @@
-import distutils
 import os
 import shutil
 import subprocess
 import sys
 
 from setuptools import find_packages, setup
+from setuptools.command.build_ext import build_ext as build_ext_orig
 
 import versioneer
 
 
-class MavenCommand(distutils.cmd.Command):
-    """Run the maven build command"""
-
-    description = "run the mvn install command"
-    user_options = []
-
-    def initialize_options(self):
-        """No options"""
-        pass
-
-    def finalize_options(self):
-        """No options"""
-        pass
+class build_ext(build_ext_orig):
```

Comment:
Was just thinking since we don't have any `Extension`s specified, this might be causing `build_ext` to not run. We could either add an empty `Extension` or move this to some other step like `build`

Reply:
Down to try out either of these approaches

> some other step like `build`

Is there a generic `build` step? Didn't see anything in [the setuptools commands](https://github.com/pypa/setuptools/tree/main/setuptools/command)

Reply:
I think [`build` comes from `distutils`]( https://github.com/pypa/setuptools/blob/6193a69d8608b6ca7e0e5ac1223d63abb55eee3b/setuptools/_distutils/command/build.py#L16 ), but `setuptools` may also alias this somewhere. Alternatively we could try using `build_py`.

Though maybe as a first step we can try adding an empty `Extension`?

Reply:
Tried this out in https://github.com/dask-contrib/dask-sql/pull/406/commits/3e5c3a95b7dbe9304e0182bcd07bf57fdd12f3f0, looks like that successfully triggers the Java build but the JAR doesn't end up getting copied successfully to its expected location

Reply:
That's good to hear 🙂

Think we need to add it to [`MANIFEST.in`]( https://github.com/dask-contrib/dask-sql/blob/ef0fa16b23cac489298c492035dcf744e7b7e351/MANIFEST.in )

Reply:
Another option might be to override the `get_outputs` method of this `build_ext` subclass to specify those file paths